lib/features/copy-paste/BpmnCopyPaste.js

```diff
@@ -69,6 +72,18 @@ export default function BpmnCopyPaste(bpmnFactory, bpmnRules, eventBus, injector
     }
   });
 
+  eventBus.on('element.copyProperty', function(context) {
+    var parent = context.parent,
+        property = context.property,
+        propertyName = context.propertyName;
+
+    if (is(parent, 'bpmn:Participant') &&
```

Comment:
Can we reformat this for better visibility:

```
if (
  abcde &&
  efasdads &&
  asdsaasdadsdas
) {
  ...
}
```

Reply:
👍